components/scream/src/physics/spa/atmosphere_prescribed_aerosol.cpp

```diff
@@ -190,6 +191,15 @@ void SPA::initialize_impl (const RunType /* run_type */)
   //       to performe_time_interpolation.
   m_buffer.spa_temp.hyam = SPAData_start.hyam;
   m_buffer.spa_temp.hybm = SPAData_start.hybm;
+
+  // Set property checks for fields in this process
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("nc_activated"),0.0,1.0e6,false);
+  // upper bound set to 1.01 as max(g_sw)=1.00757 in current ne4 data assumingly due to remapping
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("aero_g_sw"),0.0,1.01,false);
```

Comment:
I think in general asymmetry parameter can be [-1, 1].

Reply:
There are from the SPA prescribed input file. min (aer_g_sw)=0. I wonder if the definition is the same.

Reply:
Ben, you are right that in theory the aerosol asymmetry parameter can range from -1 (for 100% backscattering) to 1 (100% forward scattering).  Measurement derived and modeled values (and similarly SPA prescribed) fall within [0,1]. ne4 file however contains some overshooting (as large as 1.0076), while ne30 file are all with [0,1].

Reply:
Right but I don’t think we should set these checks to only span the values in our current input file, I think they should represent the theoretical limits. If, for example, we had something more sophisticated than SPA that actually computed values (which we will if EAMxx is ever going to provide the driver for EAMv4), then you can imagine asymmetry parameter < 0. These checks should allow for that, not just verify the current input data we have.

Reply:
The overshooting in the ne4 data is a problem we should correct as well, probably rather than fudging the bounds checks to allow input data that we know is suspect.

Reply:
Agreed but for a later date, esp. for the upper bound. Otherwise ne4 tests cannot proceed until the ne4 SPA data is updated. Also worth a thought on the purpose of this bound check in SPA. For example, since we know data generated by ne30 model are all > 0, a presence of negative value, though within theoretical limit of the asymmetry parameter, it could indicate an improper hi-order mapping (if used in certain occasion).